### ISSUE TITLE
Will this bring new capabilities to ober?

### ISSUE THREAD

**Issue by xied75** [JanJoris/ober#4]
Hi all,

Just saw this: https://blogs.windows.com/buildingapps/2016/10/17/new-windows-store-submission-api-capabilities/

I was wondering does ober need refreshing to bring these in.

Another question unrelated to this, using https://github.com/Azure/azure-sdk-for-net/ as an example, I feel the trend is to CodeGen against restful API, other than hand code, what do you guys think?

Best regards,

**Comment by JanDeDobbeleer** [comment]
Yes, it does! And I want to do it, but give me some time. it fits right in with what ober is created to do (a tool to use in a CD pipeline).

I prefer to always hand code my restful API stuff. It's not hard an leaves you with just what you need instead of (mostly) a lot of overhead.

**Comment by xied75** [comment]
@JanJoris Gently asking are we going to have some progress? :)

**Comment by JanDeDobbeleer** [comment]
I know, I know. Busy times for me right now. I'll try to do it this month, it should be rather straightforward.

**Comment by xied75** [comment]
ping...

**Comment by JanDeDobbeleer** [comment]
I know 🙁 Been rather busy lately. And my promises here seem to hold little value. I do accept PR's.

**Comment by xied75** [comment]
I'm willing to help but lacking relevant knowledge for now, don't worry. Pick up when you got free time.

**Comment by xied75** [comment]
https://github.com/Microsoft/StoreBroker

Just found this.

**Comment by JanDeDobbeleer** [comment]
Nice. I'm sorry I haven't been able to tackle this one. Not being an app dev anymore moved this really low on my priority list 😞
